membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -40,4 +47,21 @@ class AttributeController {
     onSuccess = Features.fromAttributes,
     onNotFound = Features.unauthenticated
   )
+
+  def membershipDetails = paymentDetails({t: TouchpointComponents => t.membershipPlans})
+  def digitalPackDetails = paymentDetails({t: TouchpointComponents => t.digitalPackPlans})
+
+  def paymentDetails(func: (TouchpointComponents)=>ProductFamily) = backendAction.async { implicit request =>
```

Comment:
unfortunately we can't refer to a specific instance of a ProductFamily without an HTTP request as the rate plan IDs contained within vary depending on whether you're a test user or not